tide/core/scene/Content.cpp

```diff
@@ -96,6 +96,13 @@ Content::Content(const QString& uri)
     _init();
 }
 
+Content::Content(const QString& uri, const QUuid& uuid)
+    : _uuid{uuid}
```

Comment:
different order than in ctor arg list?

Reply:
to match the declaration order, but I can invert the members if you want?

Reply:
Yeah, changing the declaration order to match the ctor arg order seems better.

Reply:
done!